sormas-ui/src/main/java/de/symeda/sormas/ui/person/PersonEditForm.java

```diff
@@ -631,6 +648,108 @@ protected void addFields() {
 			I18nProperties.getPrefixDescription(PersonDto.I18N_PREFIX, PersonDto.ADDITIONAL_DETAILS, "") + "\n"
 				+ I18nProperties.getDescription(Descriptions.descGdpr));
 		CssStyles.style(additionalDetails, CssStyles.CAPTION_HIDDEN);
+
+		hasGuardian = addField(PersonDto.HAS_GUARDIAN, CheckBox.class);
+		hasGuardian.addValueChangeListener(e -> updateHasGuardianCheckBox());
+
+		isIncapacitated = addField(PersonDto.IS_INCAPACITATED, CheckBox.class);
+		isIncapacitated.addValueChangeListener(e -> updateIsIncapacitatedCheckBox(true));
+
+		isEmancipated = addField(PersonDto.IS_EMANCIPATED, CheckBox.class);
+		isEmancipated.addValueChangeListener(e -> onEmancipatedChange());
+		hasGuardian.setEnabled(false);
+		minimumAdultAge = FacadeProvider.getConfigFacade().getMinimumAdultAge();//2 places
+		minimumEmancipatedAge = FacadeProvider.getConfigFacade().getMinimumEmancipatedAge();
+	}
+
+	private boolean canBeEmancipated() {
+		int approximateAge = getApproximateAge(ApproximateAgeType.YEARS);
+		if (approximateAge == -1){
+				return false;
+		}
+		return approximateAge >= minimumEmancipatedAge && approximateAge < minimumAdultAge;
+	}
+
+	private int getApproximateAge(ApproximateAgeType approximateAgeType) {
```

Comment:
no need to call the method with a parameter, just return always the years this is enough to be able to calculate the age you need in the calculations. Rename the method according to the changes.